src/renderers/webvr/WebVRManager.js

```diff
@@ -119,6 +106,33 @@ function WebVRManager( renderer ) {
 
 	}
 
+	function updateDrawingBufferSize() {
```

Comment:
This is the same logic previously contained in the `onVRDisplayPresentChange` handler